components/SingleReport.vue

```diff
@@ -49,11 +44,7 @@ const createLabelsFromCriteria = (currentCriterion) => {
           </AxoLink>
         </div>
         <small>{{ getVersionChangeText(props.report?.suggested_diff) }}</small>
-        <AddANote
-          :name="report.name"
-          :default-note="state[props.report.name]?.note"
-          class="mb-24"
-        />
+        <AddANote :name="report.name" class="mb-24" />
```

Comment:
Side note: Using the options API with `setup` means we can read the `defaultNote` reactively off of state directly and therefore do not need to pass it from the parent, making the component a little bit more independent.